### DISCUSSION TITLE
My GitHub Pages site is currently being built from the main branch for more than 1 week

### DISCUSSION THREAD

**Discussion by marcojunior182** [community/community#114286] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Last week I created a simple page and tried to build it on Github Pages, but so far the build has not been completed.  repository url: https://github.com/marcojunior182/portfolio
![image](https://github.com/community/community/assets/26828452/98932785-ba8f-488c-beaa-6e29ea0ccf55)

Does anyone know what it could be?

**Comment by devashish2024** [comment]
Can you try to make a new empty commit to trigger a re-build?

**Reply by marcojunior182** [reply]
Yes, I already tried to do that.

**Comment by Tgentil** [comment, marked as answer]
@marcojunior182 , I forked your repository and successfully deployed the application on GitHub Pages. To help resolve your build issue, I created a Pull Request (PR) in your repository with a minor edit to your README, which should trigger a rebuild.

Here's the link to my successful deploy: [tgentil.github.io/portfolio-marcojunior](https://tgentil.github.io/portfolio-marcojunior/)

And here's the link to the PR I created: [PR on marcojunior182/portfolio](https://github.com/marcojunior182/portfolio/pull/1)

Also, I'm attaching a screenshot of the site running and the GitHub Pages setup I used for your reference.

**Your Site Deployed on GitHub Pages:**

![Captura de tela 2024-03-28 133905](https://github.com/community/community/assets/88681055/873bd92a-216a-4c05-9930-6f61606e3753)

**My GitHub Pages Setup**

![Captura de tela 2024-03-28 133109](https://github.com/community/community/assets/88681055/15f90393-7a30-410f-9a00-1ac3c47312e1)

**Reply by marcojunior182** [reply]
thanks! I'll try to do the same here.

**Reply by Tgentil** [reply]
Great to hear that, Marco! Just a reminder, don't forget to merge the pull request I created. Alternatively, you can make a small change like adding a period at the end of your README and commit that  to force the rebuild on your GitHub Pages. This should help get your site up and running!

**Reply by marcojunior182** [reply]
Thanks! Now it's working: https://marcojunior182.github.io/portfolio/ :smile:

**Reply by andrewjjeon** [reply]
I'm having this same problem, how can I fix this?
